client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -87,7 +90,11 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 				return (
 					<Route key={ path } path={ `/${ path }` }>
 						<div className={ classnames( flow.name, flow.classnames, kebabCase( path ) ) }>
-							<ProgressBar value={ progressValue * 100 } total={ 100 } />
+							<ProgressBar
+								className={ classnames( 'flow-progress' ) }
```

Comment:
I tried to use `className="flow-progress"`, and linter now allow to commit, could you please advice on this?
<img width="1217" alt="Screenshot%20on%202022-07-27%20at%2015-31-23" src="https://user-images.githubusercontent.com/10071857/181200839-0fc435bb-7824-4493-a806-4a39eff94f7b.png">

Reply:
You can add `// eslint-disable-next-line wpcalypso/jsx-classname-namespace` in the line above `className` since   the wrapper's class name is all dynamic.

Reply:
Sorted in latest commit